CHANGES.txt

```diff
@@ -1,5 +1,6 @@
 Current
 Fixed: GITHUB-1493: Wrong exception msg when timeout on test (Krishnan Mahadevan)
+Fixed: GITHUB-1384: Huge performance issue between 6.5.2 and 6.11
```

Comment:
Just a suggestion: You should add your name there, since you put in effort on analyzing this and fixing it :)